src/routes/route-urls.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { RouteUrls } from './route-urls';
+
+const isArrayUnique = (arr: Array<string>) =>
+  Array.isArray(arr) && new Set(arr).size === arr.length;
+
+describe('RouteUrls', () => {
+  it('has all unique route paths', () => {
+    const paths = Object.values(RouteUrls);
+    // Remove the container route ('/') bc Home is being
+    // used as the nested index route
+    paths.shift();
```

Comment:
By convention `/` is at the top, but we can't be sure, filter instead

```suggestion
    // Remove the container route ('/') bc Home is being
    // used as the nested index route
    const paths = Object.values(RouteUrls).filter(url => url === '/')
```

Reply:
Yep, that makes sense ...was thinking the container route would always wrap the other routes, but that might not be the case. 👍

Reply:
Someone could def move it in the ordering ...I get that now.